src/MSBuild/Resources/Strings.resx

```diff
@@ -1033,6 +1033,37 @@
       LOCALIZATION: The prefix "MSBUILD : error MSBxxxx:" should not be localized.
     </comment>
   </data>
+  <data name="MissingGetPropertyError" UESanitized="true" Visibility="Public">
+    <value>MSBUILD : error MSB1010: Must provide a property name for the getProperty switch.</value>
```

Comment:
Where did you get these codes from? comment down at the bottom says next code assigned should be 1065.

Reply:
These error codes seem to have been skipped, so I figured I'd fill in a few gaps.

rg MSB1010 returned only my new usages. Same for MSB1014 and MSB1017. A quick google search brought up MSB1011 instead.

Reply:
In general that is not a good strategy; we should not reuse codes for obsolete errors to avoid having to rejuggle help docs or user-written things. If you're not following the policy, please mention that explicitly with justification and evidence of due diligence. The Google search you just mentioned is good evidence that this is ok.

Reply:
Note that the rg search would also find obsolete errors in the deprecated assemblies. I may be wrong, but I'm not aware of any error codes we not only stopped using but also expunged any reference to the error code anywhere in the repo.

Reply:
What I am saying is that a search in the current codebase is not sufficient. We have removed errors before. I do not think it's reliable to assume that we have _always_ made comments like

https://github.com/dotnet/msbuild/blob/69a09e4d0ad2a26fab4100e78585c85bc2f9f5a7/src/Tasks/Resources/Strings.resx#L3033-L3034